HOOKS.md

```diff
@@ -15,12 +15,14 @@ At startup Shell-operator initializes the hooks:
 
 - If hook's configuration is successful, the workqueue is filled with `onStartup` hooks.
 
- - Than, Shell-operator subscribes to Kubernetes events according to configured `kubernetes` bindings.
+- After `onStartup` hooks execution, Shell-operator subscribes to Kubernetes events according to configured `kubernetes` bindings.
+
+- Than, the work queue is filled with `kubernetes` hooks with `Synchronization` type of [binding context](#binding-context), so each hook receives all existing objects described in hook configuration.
```

Comment:
```suggestion
- Then, the work queue is filled with `kubernetes` hooks with `Synchronization` [binding context](#binding-context) type, so that each hook receives all existing objects described in hook's configuration.
```